src/Kerbero.WebApi/Controllers/GetNukiSmartLockController.cs

```diff
@@ -0,0 +1,33 @@
+using Kerbero.Domain.Common.Interfaces;
+using Kerbero.Domain.NukiActions.Models;
+using Kerbero.WebApi.ActionFilters;
+using Kerbero.WebApi.Utils.Extensions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Kerbero.WebApi.Controllers;
+
+[ApiController]
+[ServiceFilter(typeof(NukiActionFilter))]
+[Route("api/nuki/{nukiAccountId}/smartlock")]
+public class GetNukiSmartLockController: ControllerBase
+{
+	private readonly InteractorAsyncNoParam<NukiSmartLocksListPresentationDto> _nukiSmartLocksListInteractor;
+
+	public GetNukiSmartLockController(InteractorAsyncNoParam<NukiSmartLocksListPresentationDto> nukiSmartLocksListInteractor)
+	{
+		_nukiSmartLocksListInteractor = nukiSmartLocksListInteractor;
```

Comment:
```suggestion
		_nukiSmartLockListInteractor = nukiSmartLockListInteractor;
```
Probably needs renaming also the class of the interactor, should be `GetNukiSmartLockListInteractor`